tests/unit/ConversionNatureOfBusiness.spec.ts

```diff
@@ -117,9 +114,9 @@ describe('ConversionNatureOfBusiness without update', () => {
     await doneBtn.trigger('click')
 
     expect(wrapper.vm.$data.naicsText).toBe('')
-    expect(wrapper.vm.$data.onEditMode).toBeFalsy()
-    expect(wrapper.find('#naics-summary').exists()).toBeTruthy()
-    expect(wrapper.find('#naics-summary').text()).toBe('(Not entered)')
+    expect(wrapper.vm.$data.onEditMode).toBe(true)
```

Comment:
The button was clicked on line 114 but there is an error, so you're still in edit mode.